webroot/js/app.js

```diff
@@ -252,6 +270,8 @@ class Owncast {
     clearTimeout(this.disableChatTimer);
     this.disableChatTimer = null;
     this.messagingInterface.enableChat();
+
+    setInterval(this.setCurrentStreamDuration.bind(this), 1000);
```

Comment:
We should create a new timer instance up top in the constructor for this interval counter., with all the other Timers.

We should make a `this.streamDurationTimer = null` in the contructor. 
Then here `this.streamDurationTimer = setInterval...`;

Reply:
We'll have to make sure that in `handleOfflineMode()` we `clearInterval(this.streamDurationTimer)` .